mobile/Ihc/models/Credentials.js

```diff
@@ -0,0 +1,15 @@
+export default class Credentials {
+  static extractFromForm(form) {
+    const credentials = Object.assign({}, form);
+    return credentials;
+  }
+}
+
+Credentials.schema = {
+  name: 'Credentials',
+  properties: {
+    userId: 'string',
+    password: 'string',
```

Comment:
It looks like these are mostly the server changes which look fine, but in case you weren't thinking about the client side yet, if we are dealing with passwords, we should do some level of security. Ideally, this would look like:
1. the user inputs their password
2. the client hashes the password
3. the client sends the hashed password to the server
Maybe try this: https://www.npmjs.com/package/react-native-bcrypt

Reply:
Ok I will take a look. We did consider setting up better security, but for now we are just trying to get like a simple login screen.